src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/GoalResourceMapper.kt

```diff
@@ -43,5 +43,7 @@ interface GoalResourceMapper {
   @Mapping(target = "updatedBy", source = "lastUpdatedBy")
   @Mapping(target = "updatedByDisplayName", source = "lastUpdatedByDisplayName")
   @Mapping(target = "updatedAt", source = "lastUpdatedAt")
+  @Mapping(target = "createdAtPrison", constant = "BXI") // TODO RR-236 map from the domain object
```

Comment:
I've had to map the new fields with static values for now, just so the tests pass.
When I do the actual implementation this mapping will change